frontend/model/contracts/group.js

```diff
@@ -982,21 +1004,7 @@ sbp('chelonia/defineContract', {
             // new member has joined, so subscribe to their identity contract
             // TODO: Check if member is active; will be easier once profiles
             // are indexed by contract ID
-            sbp('chelonia/contract/retain', innerSigningContractID).then(() => {
-              const { profiles = {} } = state
-              const myProfile = profiles[userID]
-
-              if (isActionOlderThanUser(contractID, height, myProfile)) {
-                sbp('gi.contracts/group/emitNotificationsAfterSyncing', [], [{
-                  notificationName: 'MEMBER_ADDED',
-                  notificationData: {
-                    createdDate: meta.createdDate,
-                    groupID: contractID,
-                    memberID: innerSigningContractID
-                  }
-                }])
-              }
-            }).catch((e) => {
```

Comment:
I do not see a good reason to move this code above. Can it be reverted?

Reply:
Actually, the reason was to emit notification inside the invocation queue of syncing contracts. But since I blocked the duplicated notifications in notifications Vuex mutation, this could be reverted.